### PR TITLE
Storybook auth

### PR DESCRIPTION
**Pull request by rutvik-balar**
**Task**
Add Stories into log-in and sign-in page 

**Todo**
write stories into log in 
write stories into my sign-in screen 

**How to check**
  yarn dev
  yarn storybook 
 
**Evidences**
Screenshot 2023-12-29 at 4 46 19 PM
Screenshot 2023-12-29 at 4 46 29 PM
Screenshot 2023-12-29 at 4 46 49 PM
Screenshot 2023-12-29 at 4 46 59 PM

**build**
**Related Links**

### CODE REVIEW

**Comment by r-aritra** [comment]
pull from main to align with latest main branch

File: vite.config.mts [hunk 1]
```diff
@@ -2,7 +2,6 @@ import react from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
-// https://vitejs.dev/config/
```
Comment (r-aritra) [inline vite.config.mts:5]: do not remove auto generated comments

**Review by r-aritra — CHANGES_REQUESTED** [review]
Please remove unwanted changes from the PR

File: vite.config.mts (hunk 1 above)
Comment (rutvik-balar) [reply vite.config.mts]: resolved

**Review by r-aritra — APPROVED** [review]
lgtm

**Pull request merged**